packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageModelCamera.ts

```diff
@@ -0,0 +1,105 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import * as THREE from "three";
+
+import { PinholeCameraModel } from "@foxglove/den/image";
+
+const DEFAULT_CAMERA_STATE = {
+  near: 0.001,
+  far: 1000,
+};
+
+export class ImageModelCamera extends THREE.PerspectiveCamera {
```

Comment:
did you mean:
```suggestion
export class ImageModeCamera extends THREE.PerspectiveCamera {
```
or was this intentional?

Reply:
It was intentional, just to show its driven by the Image/Camera Model, but yeah it does look a bit confusing because the words are so close. What about `ModeledCamera`?